scripts/prettier_fix.sh

```diff
@@ -1,3 +1,8 @@
 #!/bin/bash
+export TERM=xterm-color
 
-npx prettier --write src/**/*md README.md
+SEARCH_PATH=${@:-'src/**/*.md'}
+
+npx prettier --write --loglevel warn $SEARCH_PATH
+
+git --no-pager diff --color --exit-code src/ README.md
```

Comment:
I'd suggest moving the `git diff` into the Prettier CI command, because if someone runs this locally they might well have uncommitted changes and the diff output would be unexpected.